lib/model/CAnomalyScore.cc

```diff
@@ -68,14 +68,15 @@ double scoreToProbability(double score) {
 // We use short field names to reduce the state size
 const std::string HIGH_PERCENTILE_SCORE_TAG("a");
 const std::string HIGH_PERCENTILE_COUNT_TAG("b");
-const std::string MAX_SCORE_TAG("c");
```

Comment:
would be good, to leave a comment that `c` was in use prior 6.4, otherwise someone might accidentally re-add `c`.

Reply:
Done